app/views/distribution_mailer/partner_mailer.html.erb

```diff
@@ -343,7 +343,16 @@
                   <td>
                     <p><%= @partner.name %>,</p>
                     <p>Your diaper request has been approved and you can find attached to this email a copy of the distribution you will be receiving.</p>
-                    <p>Your distribution has been set to be picked up on <%= @distribution.issued_at.strftime("%m/%d/%Y") %>.</p>
+                    <% if @distribution.issued_at_timeframe_enabled%>
+                      <p>
+                        Your distribution has been set to be picked up between <%= @distribution.issued_at.strftime("%m/%d/%Y") %> and 
+                        <%= @distribution.issued_at_end.strftime("%m/%d/%Y") %>.
```

Comment:
I think we need to change this format string to include time, because right now it'll say something like "pickup between 9/12/2020 and 9/12/2020"

Reply:
I just changed the format to include : date hour:minute am/pm ("09/12/2020 12:00am - 09/12/2020 12:30am"). Let me know if this should be different.

Reply:
Looks great